superset/security.py

```diff
@@ -110,6 +110,10 @@ def schema_access(self, datasource, user=None):
             self.can_access('schema_access', datasource.schema_perm, user=user)
         )
 
+    def has_cache_access(self, datasource):
+        # By default datasource access implies cache access.
+        return self.datasource_access(datasource)
```

Comment:
Could we make this return `True` by default. That's a no-op and the custom security manager could override this.